web/themes/custom/subtheme_videogame/src/components/node/_node--product.scss

```diff
@@ -3,7 +3,8 @@
 .product {
   &__title {
     position: relative;
-    left: 23rem;
+    display: table;
```

Comment:
Please don't use table display for aligning content. Try to use block element (`div` instead of `span`) or `display: flex` for better alignment.